client/results.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h3>Home/about/etc</h3>
+    </header>
+    <section>
+        <h1>Foogle</h1>
+    </section>
+
+    <section id="oneSearchResult">
+        <div id="website"></div>
+        <div id="link"></div>
+        <div id="summary"></div>
+    </section>
+
+    <footer>
+        <p>about the site</p>
+    </footer>
+    <script src="./scripts.js"> </script>
```

Comment:
I always put scripts here so there is nothing wrong with this, however on the other one it has it deferred at the top so I would probably do one or the other to be consistent!